online.hatsune_miku.tachidesk-vaadinui.json

```
@@ -0,0 +1,67 @@
+{
+  "app-id": "online.hatsune_miku.tachidesk-vaadinui",
+  "runtime": "org.freedesktop.Platform",
+  "runtime-version": "23.08",
+  "sdk": "org.freedesktop.Sdk",
+  "sdk-extensions": [
+    "org.freedesktop.Sdk.Extension.openjdk17"
+  ],
+  "command": "vaaui.sh",
+  "finish-args": [
+    "--env=PATH=/app/jre/bin:/app/bin/",
```

```suggestion
    "--env=PATH=/app/jre/bin:/app/bin/:/usr/bin",
```

It's preventing entering the sandbox with ` flatpak run --command=sh online.hatsune_miku.tachidesk-vaadinui`